i18n/en/en.yaml

```diff
@@ -67,9 +67,9 @@ home_show_more:
 home_show_less:
   other: Show less
 home_falco_what_is_falco_title:
-  other: Detect and respond to threats in real time
+  other: Detect threats in real time
 home_falco_what_is_falco:
-  other: Falco is the open source standard for runtime security for hosts, containers, Kubernetes and the cloud. Get real-time visibility into unexpected behaviors, config changes, intrusions, and data theft.
+  other: Falco is the open source solution for runtime security for hosts, containers, Kubernetes and the cloud. Get real-time visibility into unexpected behaviors, config changes, intrusions, and data theft.
```

Comment:
`Falco is a widely adopted open source solution ...`?

Stay with `abnormal behaviors` consistently?

proposed rewrite: 

`Get real-time visibility into abnormal behaviors and potential security threats, intrusions, and data theft or compliance violations.`

Reply:
added thanks!